css/components/Button.css

```diff
@@ -0,0 +1,65 @@
+@import "../style.css";
+
+.btn{
+    border: 1px solid;
+    margin: 10px;
```

Comment:
rem always better choice

Reply:
but below 10px and 10px we can't use rem, Vikram sir said